.chloggen/move-attributes.yaml

```diff
@@ -0,0 +1,27 @@
+# Use this changelog template to create an entry for release notes.
+
+# One of 'breaking', 'deprecation', 'new_component', 'enhancement', 'bug_fix'
+change_type: breaking
```

Comment:
Is it breaking, given that the feature gate is in alpha? Feel free to submit a PR changing it to `enhancement` if you agree.